data/json/items/generic.json

```diff
@@ -3675,7 +3675,7 @@
     "color": "brown",
     "name": { "str": "training dummy" },
     "category": "other",
-    "description": "A hand-made humanoid figure, useful for training in close quarters combat.  This one is made from wood and it has some duct tape to mark the striking points.",
+    "description": "A hand-made humanoid figure, useful for training in close quarters combat.  This one is made from wood and it has some duct tape to mark the striking points.  You need to embed it to the ground for train.",
```

Comment:
Suggestion: train->training
Question: Should "embed it to" be "embed it into"? or maybe just reworded because I don't think it reads well.